### PR TITLE
eslint fix packages showcase

### PR DESCRIPTION
**Pull request by matthieu-crouzet**

### CODE REVIEW

**Review by mrednic-1A — APPROVED** [review]

File: apps/showcase/src/app/rules-engine/rules-engine.template.html [hunk 1]
```diff
@@ -32,7 +32,7 @@ <h2 id="rules-engine-example">Example</h2>
       <a href="https://github.com/AmadeusITGroup/otter/blob/main/apps/showcase/src/components/showcase/rules-engine" target="_blank" rel="noopener">Source code</a>
       <ul ngbNav #nav="ngbNav" class="nav nav-tabs mt-3">
         <li ngbNavItem class="nav-item" role="presentation">
-          <a ngbNavLink class="nav-link" [class.active]="activeRuleTab === 'configuration'" (click)="activateRuleTab('configuration')">Configuration rule</a>
+          <a tabindex="0" ngbNavLink class="nav-link" [class.active]="activeRuleTab === 'configuration'" (click)="activateRuleTab('configuration')" (keyup.enter)="activateRuleTab('configuration')">Configuration rule</a>
```
Comment (cpaulve-1A) [inline apps/showcase/src/app/rules-engine/rules-engine.template.html:35]: Do we really need all of this for a link markdown?

Comment (cpaulve-1A) [inline apps/showcase/src/app/rules-engine/rules-engine.template.html:35]: We should not need all of this, we should follow the design factory tabs recommendation as it should be accessible.

File: apps/showcase/src/components/showcase/basic/basic-pres.template.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="card my-3">
   <div class="row g-0">
     <div class="col-4 bg-body-tertiary d-flex align-items-center justify-center">
-      <img src="assets/otter.svg" class="img-fluid rounded-start" aria-hidden="true">
+      <img alt="Otter image" src="assets/otter.svg" class="img-fluid rounded-start" aria-hidden="true">
```
Comment (cpaulve-1A) [inline apps/showcase/src/components/showcase/basic/basic-pres.template.html:4]: Maybe we could add some context on why we have this otter here?
```suggestion
      <img alt="Otter image that may be replaced depending on user inputs" src="assets/otter.svg" class="img-fluid rounded-start" aria-hidden="true">
```

File: apps/showcase/src/components/showcase/dynamic-content/dynamic-content-pres.template.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <div class="card my-3">
   <div class="row g-0">
     <div class="col-4 bg-body-tertiary d-flex align-items-center justify-center">
-      <img [src]="'otter-dynamic.svg' | o3rDynamicContent" class="img-fluid rounded-start" aria-hidden="true">
+      <img alt="Otter image" [src]="'otter-dynamic.svg' | o3rDynamicContent" class="img-fluid rounded-start" aria-hidden="true">
```
Comment (cpaulve-1A) [inline apps/showcase/src/components/showcase/dynamic-content/dynamic-content-pres.template.html:4]:
```suggestion
      <img alt="An otter which will be replaced with another image to demonstrate the dynamic content capabilities" [src]="'otter-dynamic.svg' | o3rDynamicContent" class="img-fluid rounded-start">
```

File: apps/showcase/src/components/showcase/rules-engine/rules-engine-pres.template.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <div class="card my-3">
   <div class="row g-0">
     <div class="col-4 bg-body-tertiary d-flex align-items-center justify-content-center">
-      <img [src]="'otter.svg' | o3rDynamicContent" class="img-fluid rounded-start" aria-hidden="true">
+      <img alt="Otter image" [src]="'otter.svg' | o3rDynamicContent" class="img-fluid rounded-start" aria-hidden="true">
```
Comment (cpaulve-1A) [inline apps/showcase/src/components/showcase/rules-engine/rules-engine-pres.template.html:4]:
```suggestion
      <img alt="An otter image which will be modified depending on the user search inputs" [src]="'otter.svg' | o3rDynamicContent" class="img-fluid rounded-start">
```

**Review by fpaul-1A — APPROVED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Review by kpanot — APPROVED** [review]

**Pull request merged**